Gemfile

```diff
@@ -0,0 +1,56 @@
+source 'https://rubygems.org'
+
+git_source(:github) do |repo_name|
+  repo_name = "#{repo_name}/#{repo_name}" unless repo_name.include?("/")
+  "https://github.com/#{repo_name}.git"
+end
+
+
+# Bundle edge Rails instead: gem 'rails', github: 'rails/rails'
+gem 'rails', '~> 5.0.2'
```

Comment:
Ohhh that's why some of this looks weird to me, shiny new rails! ✨  (we used 4.2)